Sorry: IndentationError: unexpected indent (image_processor.py, line 172)
